charts/zabbix/README.md

```diff
@@ -484,11 +488,12 @@ The following tables lists the configurable parameters of the chart and their de
 | zabbixweb.readinessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
 | zabbixweb.replicaCount | int | `1` | Number of replicas of ``zabbixweb`` module |
 | zabbixweb.resources | object | `{}` | Requests and limits of pod resources. See: [https://kubernetes.io/docs/concepts/configuration/manage-resources-containers](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers) |
-| zabbixweb.service | object | `{"annotations":{},"clusterIP":null,"port":80,"type":"ClusterIP"}` | Certificate containing certificates for SAML configuration saml_certs_secret_name: zabbix-web-samlcerts |
+| zabbixweb.service | object | `{"annotations":{},"clusterIP":null,"nodePort":31080,"port":80,"type":"NodePort"}` | Certificate containing certificates for SAML configuration saml_certs_secret_name: zabbix-web-samlcerts |
 | zabbixweb.service.annotations | object | `{}` | Annotations for the zabbix-web service |
 | zabbixweb.service.clusterIP | string | `nil` | Cluster IP for Zabbix web |
+| zabbixweb.service.nodePort | int | `31080` | NodePort port to allocate (only if service.type = NodePort) |
 | zabbixweb.service.port | int | `80` | Port to expose service |
-| zabbixweb.service.type | string | `"ClusterIP"` | Type of service for Zabbix web |
+| zabbixweb.service.type | string | `"NodePort"` | Type of service for Zabbix web |
```

Comment:
Same as above regarding NodePort service

Reply:
Hi @sa-ChristianAnton!

To use ingress, the service must be NodePort. I switched to make it easier. But I understand your concerns and have rolled back.